### DISCUSSION TITLE
集群外部无法访问sealos 创建的k8s集群 集群内部可以

### DISCUSSION THREAD

**Discussion by Fleezesd** [labring/sealos#3382] [Q&A]
这边部署集群时 出现了集群外部无法访问内部服务的问题 

可能是因为lvscare 相关联的pod 被删除掉了 在kube-system没有看见lvscare相关联的pod

现在有对应的lvs部署的yaml文件么 因为很多其他自己自定义的配置 比如istio 包括一些服务都部署好了 不想重新删除集群再部署一遍

**Comment by cuisongliu** [comment]
> 这边部署集群时 出现了集群外部无法访问内部服务的问题
> 
> 可能是因为lvscare 相关联的pod 被删除掉了 在kube-system没有看见lvscare相关联的pod
> 
> 现在有对应的lvs部署的yaml文件么 因为很多其他自己自定义的配置 比如istio 包括一些服务都部署好了 不想重新删除集群再部署一遍

nodeport访问不了还是怎么回事，lvscare的pod是静态pod除非删除文件，一般运行在node节点上
